var/spack/repos/builtin/packages/coreneuron/package.py

```diff
@@ -19,12 +19,12 @@ class Coreneuron(CMakePackage):
     git      = "https://github.com/BlueBrain/CoreNeuron"
 
     version('develop', branch='master', submodules=True)
+    version('0.19', tag='0.19', submodules=True, preferred=True)
```

Comment:
Ohey!! big bump! 🎉 a bit afraid of this, but as long as we have enough time to test :)

Reply:
Jorge has tested all sims. So let’s trust him this time!